src/rollup/types.d.ts

```diff
@@ -487,8 +491,10 @@ export interface SerializedTimings {
 }
 
 export interface OutputAsset {
+	type: 'asset';
 	code?: undefined;
```

Comment:
I believe this property was only there for type checks and can be removed safely